src/components/StyledComponents/PageContentWrapper.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+const PageContentWrapper = styled.div`
```

Comment:
My initial reaction to this is yuck!  Why?  I worked for Steve Champeon, who coined the phrase "progressive enhancement", so this seems like a fundamental separation of concerns issue.  That said, I do realize React is significantly blurring the separation of concerns lines, but it seems like this is going all in on it.

Additionally, it seems like hac-dev and dynamic-plugin-sdk-utils are using external styles, so I think it's better to use one approach for consistency.

Reply:
The ES 6 syntax can be jarring if you aren't used to it.  There is a way to pass a JSON of styles, but you don't see that used too much because who really likes the scss code in json format?

Here is where I think we should move forward.  I think the use of styled components (or something similar) would be extremely helpful.  It might be obvious in my simple example but actually leads to a better separation of structure and styling.  I'll revert this to an external CSS file, but I can present about styled-components in an upcoming meeting so the benefits can be more fully shown and discussed.

Reply:
I think is a great discussion - we should probably move this to a separate PR and have a discussion with a broader team. I personally have used https://cssinjs.org/?v=v10.9.2 on previous teams - and it worked wonders to prevent styles from bleeding outside of components un-intentionally.